app/src/test/java/com/rickyhu/hushkeyboard/ui/SettingsScreenUiTest.kt

```diff
@@ -30,8 +34,12 @@ class SettingsScreenUiTest {
                 state = SettingsState(),
                 onThemeSelected = {},
                 onWideNotationOptionSelected = {},
-                onAddSpaceBetweenNotationChanged = {},
-                onVibrateOnTapChanged = {}
+                onAddSpaceBetweenNotationChanged = {
+                    addSpaceAfterNotationSwitchValue = it
+                },
+                onVibrateOnTapChanged = {
+                    vibrateOnTapSwitchValue = it
+                }
```

Comment:
Help me replace `it` with a named variable for better readability.

```suggestion
                onAddSpaceBetweenNotationChanged = { value ->
                    addSpaceAfterNotationSwitchValue = value
                },
                onVibrateOnTapChanged = { value ->
                    vibrateOnTapSwitchValue = value
                }
```